pointcept/engines/train.py

```diff
@@ -263,7 +269,7 @@ def build_train_loader(self):
             collate_fn=partial(point_collate_fn, mix_prob=self.cfg.mix_prob),
             pin_memory=True,
             worker_init_fn=init_fn,
-            drop_last=True,
+            drop_last=len(train_data) > self.cfg.batch_size_val_per_gpu,
```

Comment:
This was causing issues during our overfitting experiments.

Reply:
@Streakfull 
For train_loader, I think here should be `elf.cfg.batch_size_per_gpu` instead of `self.cfg.batch_size_val_per_gpu`?